.github/workflows/deploy-carbonplan.yaml

```diff
@@ -0,0 +1,30 @@
+name: Deploy and Test carbonplan hub
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - deployer/**
+      - hub-templates/**
+      - requirements.txt
+      - dev-requirements.txt
+      - config/secrets.yaml
+      - config/hubs/carbonplan.cluster.yaml
```

Comment:
Is handling this "run on changes to a specific cluster.yaml file" the crux of the reason we are using workflow files per cluster rather than a matrix job?

Reply:
@sgibson91 It wasn't intentional - i was just following the pattern @GeorgianaElena set up. It is nice to not run them when there's no changes though.

Reply:
For sure! I was going to suggest using something like this action so set a conditional https://github.com/dorny/paths-filter
So we maintain the same behaviour of not doing a deploy when nothing's changed, but we can consolidate into a single workflow file and new clusters can just be added to the matrix element list.

Reply:
(This shouldn't block the PR btw - merge away and we can open an issue to track this discussion if you like)